dll/win32/msvcrt/CMakeLists.txt

```diff
@@ -1,45 +1,242 @@
 
-include_directories(${REACTOS_SOURCE_DIR}/sdk/lib/crt/include)
-spec2def(msvcrt.dll msvcrt.spec ADD_IMPORTLIB)
+###############################################################################
+# Global compile options
+###############################################################################
 
-# Let consumers of msvcrt have the right defines
-target_compile_definitions(libmsvcrt INTERFACE _DLL __USE_CRTIMP)
-# Embed msvcrtex into libmsvcrt
-target_sources(libmsvcrt PRIVATE $<TARGET_OBJECTS:msvcrtex>)
+# Use conforming swprintfs
+remove_definitions(-D_CRT_NON_CONFORMING_SWPRINTFS)
 
-# Embed RTC libs
-if (STACK_PROTECTOR)
-    target_sources(libmsvcrt PRIVATE $<TARGET_OBJECTS:gcc_ssp_msvcrt>)
-    target_link_libraries(libmsvcrt INTERFACE libkernel32) # For OutputDebugStringA
-endif()
+# This requires NT6
+remove_definitions(-D_WIN32_WINNT=0x502 -DWINVER=0x502)
+add_definitions(-D_WIN32_WINNT=0x601 -DWINVER=0x601)
+
+# Hack to replace the old CRT include directory with the msvcrt include directory
+get_property(INCLUDE_DIRS DIRECTORY . PROPERTY INCLUDE_DIRECTORIES)
+list(REMOVE_ITEM INCLUDE_DIRS "${REACTOS_SOURCE_DIR}/sdk/include/crt")
```

Comment:
This is fairly brittle, if the path is not exactly formatted like you do here (e.g. trailing slash, some `..` crap or so in it) this will fail....

Reply:
So better nobody put any crap into that global include dir...
I'd love to get rid of it and replace it with CMake interface definitions, but that is a a ton of work.